x-pack/legacy/plugins/siem/public/components/drag_and_drop/draggable_wrapper.tsx

```diff
@@ -47,34 +47,50 @@ const ProviderContentWrapper = styled.span`
   }
 `;
 
+type RenderFunctionProp = (
+  props: DataProvider,
+  provided: DraggableProvided,
+  state: DraggableStateSnapshot
+) => React.ReactNode;
+
 interface OwnProps {
   dataProvider: DataProvider;
   inline?: boolean;
-  render: (
-    props: DataProvider,
-    provided: DraggableProvided,
-    state: DraggableStateSnapshot
-  ) => React.ReactNode;
+  render: RenderFunctionProp;
   truncate?: boolean;
 }
 
-type Props = OwnProps & PropsFromRedux;
+type Props = OwnProps;
 
 /**
  * Wraps a draggable component to handle registration / unregistration of the
  * data provider associated with the item being dropped
  */
 
-const DraggableWrapperComponent = React.memo<Props>(
-  ({ dataProvider, registerProvider, render, truncate, unRegisterProvider }) => {
+export const DraggableWrapper = React.memo<Props>(
```

Comment:
seeing a sometimes-reproducible issue when adding an `AND` clause:

![and-clause](https://user-images.githubusercontent.com/4459398/75397591-7cd65400-58b4-11ea-9583-e164becc3936.gif)